public/less/angle/app/top-navbar.less

```diff
@@ -61,29 +61,29 @@
     position: relative;
     // overlaps nav-wrapper
     z-index: 11;
-    
+
```

Comment:
Why is there a z-index 11, here. Do you think it would be worth it, to document why? If not, leave it as it is.